src/DataGridBundle/Resources/public/js/grid.js

```diff
@@ -52,7 +52,7 @@ define([
                 var gridOptions = {
                     collection: collection,
                     className: 'backgrid table table-bordered table-hover',
-                    emptyText: "no data"
+                    emptyText: "There currently is no data to display."
```

Comment:
`is` should be `are` here

```suggestion
                    emptyText: "There currently are no data to display."
```